packages/management-api/src/managed-pod.ts

```diff
@@ -108,6 +121,36 @@ export class ManagedPod {
     return this._management
   }
 
+  getManagementError(): Error | null {
+    if (!this._management.status.error) return null
+
+    return new Error(`(${this._management.status.error.code}): ${this._management.status.error.message}`)
+  }
+
+  private setManagementError(errorCode: number, errorMsg: string) {
+    this._management.status.error = {
+      code: errorCode,
+      message: errorMsg,
+    }
+  }
+
+  get errorPolling(): ErrorPolling {
+    return this._errorPolling
+  }
+
+  resetErrorPolling() {
+    this._errorPolling.count = 0
+    this._errorPolling.threshold = 1
+  }
+
+  incrementErrorPollCount() {
+    this._errorPolling.count = ++this._errorPolling.count
```

Comment:
Can't we just make it like this?  It looks a bit redundant.
```
this._errorPolling.count++
```

Reply:
:-)
Yes we can. Although, I ran into trouble with that syntax as I started doing `const count = this._errorPolling.count++` and count remained as 0! So, I think I was just being over-zealous in ensuring the value was indeed incremented.